### ISSUE TITLE
state space representation

### ISSUE THREAD

**Issue by aceeccc** [openconnectome/Statistical-Connectomics-Class#187]
Is it correct to say that the A matrix from the state space model seems to represent a weighted adjacency matrix? In this case, can we compute centrality and other network analysis things off of this matrix? Also, how computationally expensive is it to build the A matrix out of a set of fMRI data?

**Comment by mrjiaruiwang** [comment]
That's an interesting proposition. I guess if the adjacency matrix represents the connections in the brain, and if the connections are what you define as the state of the brain, then yes either A or a linear transformation of A should be okay for use in a state space model. It helps if A is invertible, which for undirected graphs it is.

**Comment by akim1** [comment]
The weighting on an adjacency matrix are required to be between 0 and 1. Elements in an A matrix don't have to strictly between 0 and 1. So I think the answer is no.

**Comment by adjordan** [comment]
I don't think that it would be computationally expensive, at least not more than using other data types.
